retroarch.cfg

```diff
@@ -261,23 +248,20 @@
 
 # Offset for where messages will be placed on screen. Values are in range 0.0 to 1.0 for both x and y values.
 # [0.0, 0.0] maps to the lower left corner of the screen.
-# video_message_pos_x = 0.05
-# video_message_pos_y = 0.05
+# video_msg_pos_x = 0.05
+# video_msg_pos_y = 0.05
 
-# Color for message. The value is treated as a hexadecimal value.
-# It is a regular RGB hex number, i.e. red is "ff0000".
-# video_message_color = ffffff
+# Color for message.
+# video_msg_color_r =
+# video_msg_color_g =
+# video_msg_color_b =
```

Comment:
What are the default values for `video_msg_color`?

Reply:
@RobLoach I intentionally changed it to `video_msg_color`. [According to configuration.h](https://github.com/libretro/RetroArch/blob/master/configuration.h#L274-L279), the variable names were changed at some point.

Reply:
@RobLoach I'll revert the variable names for now, since they could be considered outside the scope of my PR.